library/init/meta/interactive.lean

```diff
@@ -66,6 +81,7 @@ private meta def parser_desc_aux : expr → tactic (list format)
 | `(ident)  := return ["id"]
 | `(ident_) := return ["id"]
 | `(qexpr) := return ["expr"]
+| `(location) := return ["loc?"]
```

Comment:
The default description complained that it did not know how to display `loc.wildcard`. Can I get that with a `to_string` method? If so, I'll try it and see how it looks.